### DISCUSSION TITLE
Skyline has panicked Error Code: 2--0069

### DISCUSSION THREAD

**Discussion by SealMeat** [Raytwo/ARCropolis#327] [Issues]
Not sure where to ask this but skyline panics and gives this error. Don't know what to do so any help is appreciated.
![2022042614135400-1AB131B6E6571375B79964211BB3F5AE](https://user-images.githubusercontent.com/104461335/166088785-d29959c4-d728-4c85-af6a-f4a27843502b.jpg)

**Comment by Raytwo** [comment]
One of the configuration file has been corrupted or broken, consider deleting ``sd:/ultimate/arcropolis/config``
